frontend/src/components/Header.tsx

```diff
@@ -160,23 +160,28 @@ export default function Header() {
               {link.text}
             </NavLink>
           ))}
-          <NavButton
-            href="https://github.com/OWASP/Nest"
-            defaultIcon={faRegularStar}
-            hoverIcon={faSolidStar}
-            defaultIconColor="text-white"
-            hoverIconColor="text-yellow-400"
-            text="Star"
-          />
 
-          <NavButton
-            href="https://owasp.org/donate/?reponame=www-project-nest&title=OWASP+Nest"
-            defaultIcon={faRegularHeart}
-            hoverIcon={faSolidHeart}
-            defaultIconColor="#b55f95"
-            hoverIconColor="#d9156c"
-            text="Sponsor"
-          />
+          <div
```

Comment:
Can you update the `star` icon outline color to be yellow as we discussed? Same as `heart` icon is pink by default.
Also, could you move the buttons all the way to the bottom of the menu?
![Screenshot 2025-02-23 at 11 32 37 AM](https://github.com/user-attachments/assets/140ef5b3-b574-4f13-81c7-ccaa04df6561)